### ISSUE TITLE
Radio Group Component has issues with array of objects

### ISSUE THREAD

**Issue by karaeren** [alfonsobries/vue-tailwind#64]
My radio group element is declared like this:

```
<t-radio-group
  :options="[
    { value: 1, text: 'ABC' },
    { value: 2, text: 'XYZ' }
  ]"
  v-model="modelVal"
></t-radio-group>
```

The modelVal is set to 1 in the data part. The expected behavior is to change the modelVal in the data to the value of the option. However, it doesn't change it and also, it doesn't uncheck the previously checked radio. So you end up with both radios checked and false modelVal...

This is how it renders
```
<div class="px-4 py-2 flex flex-col">
   <label for="1" class="">
      <span class=""><input value="1" id="1" type="radio" checked="checked" class="form-radio"></span>
      <span class="ml-2 text-gray-700 text-sm">ABC</span>
   </label>
   <label for="2" class="">
      <span class=""><input value="2" id="2" type="radio" class="form-radio"></span>
      <span class="ml-2 text-gray-700 text-sm">XYZ</span>
   </label>
</div>
```

**Comment by karaeren** [comment]
Simply adding a name to the t-radio-group seems to fix the multiple checked radio issue. However, it still does not change the data...

**Comment by alfonsobries** [comment]
@karaeren Thanks for the report already fixed the issue, just upgrade to ^1.2.1
